### ISSUE TITLE
GoR dashboard & leaderboard

### ISSUE THREAD

**Issue by moul** [gnolang/gno#501]
We'll need to develop a dynamic dashboard/leaderboard.
Ideally, it should be 100% produced as a gno realm.

Do you have any ideas/suggestions on what we could add or how we could display the info?

**Comment by zivkovicmilos** [comment]
>Do you have any ideas/suggestions on what we could add or how we could display the info?

I'm guessing we'd use a max heap for tracking the users (priority queue) by some metric and a simple Markdown table that gets filled up in the `Render` method of the realm. 

What kind of info do we need to display here?

**Comment by moul** [comment]
> We'd use a max heap for tracking the users (priority queue) by some metric and a simple Markdown table that gets filled up in the Render method of the realm.

Yes, I suggest we start with something simple like this, then iterate.

> What kind of info do we need to display here?

my list (to be completed):

* global stats [amount of contributors, contributions].
* top contributors [by points?, by contributions?].
* recent contributions (reviewed and to-review).
* if we find a way to show "you're here" in the competition, i.e., "80% of phase 1"; it would be awesome and useful.
